packages/components/src/template-registry.ts

```diff
@@ -29,4 +30,10 @@ export default interface HdsComponentsRegistry {
   // Helpers
   'hds-link-to-models': typeof HdsLinkToModelsHelper;
   'hds-link-to-query': typeof HdsLinkToQueryHelper;
+  HdsBadgeComponent: typeof HdsBadgeIndexComponent;
+
```

Comment:
```suggestion
  // Badge
  'Hds::Badge': typeof HdsBadgeIndexComponent;
  'hds/badge': typeof HdsBadgeIndexComponent;
  'HdsBadge': typeof HdsBadgeIndexComponent;
  HdsBadgeComponent: typeof HdsBadgeIndexComponent;
```

Reply:
just guessing, we might consider some order, like components, then helpers 🤔

Reply:
If it can help, in terms of mental model and organization, we tend to follow the same order on the website as well as on the showcase: foundations → components → layouts → overrides → utilities (alphabetical within each group)